### PR TITLE
feat(ui): Added borders

### PR DESCRIPTION
**Pull request by bryant-the-coder**

### CODE REVIEW

File: lua/duckytype/settings.lua [hunk 1]
```diff
@@ -1,29 +1,62 @@
...
+-- 	{ "═", "FloatBorder" },
+-- 	{ "╗", "FloatBorder" },
+-- 	{ "║", "FloatBorder" },
+-- 	{ "╝", "FloatBorder" },
+-- 	{ "═", "FloatBorder" },
+-- 	{ "╚", "FloatBorder" },
+-- 	{ "║", "FloatBorder" },
+-- }
+
+local border = {
```
Comment (kwakzalver) [inline lua/duckytype/settings.lua:23]: border = 'rounded'

File: lua/duckytype/settings.lua [hunk 2]
```diff
@@ -1,29 +1,62 @@
...
+--     { "━", "FloatBorder" },
+--     { "┓", "FloatBorder" },
+--     { "┃", "FloatBorder" },
+--     { "┛", "FloatBorder" },
+--     { "━", "FloatBorder" },
+--     { "┗", "FloatBorder" },
+--     { "┃", "FloatBorder" },
+-- }
+
+-- local border = {
```
Comment (kwakzalver) [inline lua/duckytype/settings.lua:12]: border = 'double'

File: lua/duckytype/settings.lua [hunk 3]
```diff
@@ -1,29 +1,62 @@
...
```
Comment (kwakzalver) [inline lua/duckytype/settings.lua:1]: border = 'single'

**Review by kwakzalver — COMMENTED** [review]
these border styles are already supported by default `:h nvim_open_win` then scroll down to **Parameters:** and check out the `{config}` for the default border settings

**Comment by bryant-the-coder** [comment]
Ohh

**Pull request closed without merging**